src/main/java/org/commcare/session/SessionInstanceBuilder.java

```diff
@@ -88,14 +88,15 @@ private static String getCalloutSearchResultCount(StackFrameStep step) {
 
     private static void addMetadata(TreeElement sessionRoot, String deviceId,
                                     String appversion, String username,
-                                    String userId, long drift) {
+                                    String userId, long drift, String windowWidth) {
         TreeElement sessionMeta = new TreeElement("context", 0);
 
         addData(sessionMeta, "deviceid", deviceId);
         addData(sessionMeta, "appversion", appversion);
         addData(sessionMeta, "username", username);
         addData(sessionMeta, "userid", userId);
         addData(sessionMeta, "drift", String.valueOf(drift));
+        addData(sessionMeta, "windowwidth", windowWidth);
```

Comment:
`windowwidth` troubles me a bit, can we underscore it as `window_width` although inconsistent with other params.

Reply:
sure, I'll change it to window_width